### DISCUSSION TITLE
Your connection isn't private

### DISCUSSION THREAD

**Discussion by colonical** [cockpit-project/cockpit#17538] [General]
Hi!

I tried to login to my cockpit on 9090 port, but doesnt work, i get this message in my browser. Something ssl certificate problem maybe? :/
i also installed to another site , there it goes but also only with http not https.
Both server can receive ssl i guess i skip 1 step somewhere in config where i can set it up. 
Could you help me please.

**Comment by garrett** [comment]
You haven't provided enough details for us to know what you're encountering.

Please share more details, such as:

- the text of the error message (or a screenshot of it)
- which version of Linux you're installing Cockpit on
- your browser (and browser version)

Additionally:

- Are you using a self-signed certificate (default) or a custom certificate?
- Are you using a proxy?
- Are you connecting from a different machine from the server?
- Are you connecting through the local network or over the Internet?

Relevant documentation:

- SSL/TLS Usage: https://cockpit-project.org/guide/latest/https
- Cockpit Authentication: https://cockpit-project.org/guide/latest/authentication
- If you're trying to use a proxy:
  - Apache: https://cockpit-project.org/external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt
  - NGINX: https://cockpit-project.org/external/wiki/Proxying-Cockpit-over-NGINX
- Cockpit deployment guide: https://cockpit-project.org/guide/latest/guide
